### PR TITLE
SOX-121 Create list of changed things when MR put to review

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira: SOX-121
KBC: https://github.com/keboola/connection/pull/4508

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

**There is BC break in experimental MR feature, that changes "content" from "json string" to "object"**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -8958,7 +8958,7 @@ Detail of Merge request also contains `content` of changed objects.
             "title": "Refactor DB extractors",
             "description": "DB exctractor have new columns now",
             "state": "in_review",
-            "content": "",
+            "content": {},
```
Comment (jirkasemmler) [inline apiary.apib:8961]: toto se musi upravit u vsech odpovedi MR endpointu. Bohuzel se mi to v rychlosti nepovedlo presunout do Data structures, abychom to pouzivali pres referenci. Tak bud to zkus a uprav to pak na jednom miste, nebo to prosim prepis vsude.

Zaroven si ale myslim, ze minimalne u odpovedi [request-review](https://keboola.docs.apiary.io/#reference/development-branches/merge-requests/put-mr-in-review) by tady nemel byt `{}`, ale nejaky obsah, aby z tohoto examplu bylo patrne co to vraci . A nemusel si to konzument provolavat

File: tests/Backend/SOX/MergeRequestsTest.php [hunk 2]
```diff
@@ -730,6 +731,135 @@ public function testConfigurationUpdatedInBranch(): void
...
+        $mrId = $this->developerClient->createMergeRequest([
+            'branchFromId' => $newBranch['id'],
+            'branchIntoId' => $defaultBranch['id'],
+            'title' => $this->generateUniqueNameForString('mr-title'),
+            'description' => $this->generateUniqueNameForString('mr-description'),
+        ]);
+
+        $reviewerClient = $this->getReviewerStorageApiClient();
+        $this->developerClient->mergeRequestRequestReview($mrId);
+        $mr = $reviewerClient->getMergeRequest($mrId);
```
Comment (jirkasemmler) [inline tests/Backend/SOX/MergeRequestsTest.php:815]: proc to tady musi getovat ten reviewer? jakoze asi to nicemu nevadi, ale unika mi neco?

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jen prosim pridej example do apiary. Jinak cajk

Je super, ze jsi tam tech verzi udelal vic, aby to bylo videt

File: tests/Backend/SOX/MergeRequestsTest.php (hunk 2 above)
Comment (tomasfejfar) [reply tests/Backend/SOX/MergeRequestsTest.php:815]: Jen copy paste odjinud

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**